rubert/benches/matmul.rs

```diff
@@ -0,0 +1,115 @@
+//! Run as `cargo bench --bench matmul --features bench`.
+
+use std::{fs::File, io::BufReader};
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use ndarray::Array2;
+use onnxruntime::{environment::Environment, GraphOptimizationLevel, LoggingLevel};
+use tract_onnx::prelude::{tvec, Datum, Framework, InferenceFact, InferenceModelExt};
+
+macro_rules! bench_tract {
```

Comment:
This and `bench_tract` could be a function `(manager: &mut Context, name: &str, model: &str)`.